packages/webdriverio/src/commands/browser/setCookies.js

```diff
@@ -26,11 +26,11 @@
  * @param {Object} cookie cookie object
  * @param {String} cookie.name The name of the cookie
  * @param {String} cookie.value The cookie value
- * @param {String} cookie.path  (Optional) The cookie path
- * @param {String} cookie.domain  (Optional) The domain the cookie is visible to
- * @param {Boolean} cookie.secure (Optional) Whether the cookie is a secure cookie
- * @param {Boolean} cookie.httpOnly  (Optional) Whether the cookie is an httpOnly cookie
- * @param {Number} cookie.expiry  (Optional) When the cookie expires, specified in seconds since midnight, January 1, 1970 UTC
+ * @param {String=} cookie.path  (Optional) The cookie path
+ * @param {String=} cookie.domain  (Optional) The domain the cookie is visible to
+ * @param {Boolean=} cookie.secure (Optional) Whether the cookie is a secure cookie
+ * @param {Boolean=} cookie.httpOnly  (Optional) Whether the cookie is an httpOnly cookie
+ * @param {Number=} cookie.expiry  (Optional) When the cookie expires, specified in seconds since midnight, January 1, 1970 UTC
```

Comment:
All of these cookie params are still not valid I don't think. ```setCookies``` only takes one parameter as an object.

Reply:
you are right, having
```js
browser.setCookies(cookie, cookie.name, cookie.value, cookie.path, cookie.domain, cookie.secure, cookie.httpOnly, cookie.expiry)
```
would be wrong. I will update with the example from your PR